nifi-nar-bundles/nifi-ccda-bundle/nifi-ccda-processors/src/main/resources/docs/org.apache.nifi.processors.ccda.ExtractCCDAAttributes/additionalDetails.html

```diff
@@ -0,0 +1,108 @@
+          "codeSystem": "2.16.840.1.113883.6.96",
+          "codeSystemName": "SNOMED CT",
+          "displayName": "Angina (disorder)",
+          "translations": [
+            {
+              "code": "413.9",
+              "codeSystem": "2.16.840.1.113883.6.103",
+              "codeSystemName": "ICD-9CM (diagnosis codes)",
+              "displayName": "Other and unspecified angina pectoris"
+            }
+          ]
+        },
+        "statusCode": {
+          "code": "completed"
+        },
+        "effectiveTime": {
+          "low": "20130711"
+        }
+      }
+    ]
+            </code></pre>
+        </p>
+        <h3>Example Attribute Output</h3>
+        <p>
+            <pre><code>
+    problemSection.act_04.observation.effectiveTime.low=20130711
+    problemSection.act_04.observation.id.extension=121387536
+    problemSection.act_04.observation.id.root=2.16.840.1.113883.1.13.99999.999239
+    problemSection.act_04.observation.statusCode.code=completed
+    problemSection.act_04.observation.values.code=194828000
+    problemSection.act_04.observation.values.codeSystem=2.16.840.1.113883.6.96
+    problemSection.act_04.observation.values.codeSystemName=SNOMED CT
+    problemSection.act_04.observation.values.displayName=Angina (disorder)
+    problemSection.act_04.observation.values.translations.code=413.9
+    problemSection.act_04.observation.values.translations.codeSystem=2.16.840.1.113883.6.103
+    problemSection.act_04.observation.values.translations.codeSystemName=ICD-9CM (diagnosis codes)
+    problemSection.act_04.observation.values.translations.displayName=Other and unspecified angina pectoris
+    problemSection.act_04.statusCode.code=active
+            </code></pre>
+        </p>
+        <h3>Example Parser Mapping</h3>
+        <p>
+            This processor is driven by a mapping file which specifies the element relationships. For example
+            <pre><code>
+    org.openhealthtools.mdht.uml.cda.consol.impl.ProblemObservationImpl=id#element.ids\
+        @values#element.values[0]\
+        @statusCode#element.statusCode\
+        @effectiveTime#element.effectiveTime\
+        @negation#element.negationInd\
+        @problemStatus#element.problemStatus
+    org.openhealthtools.mdht.uml.cda.consol.impl.ProblemStatusImpl=id#element.id\
+        @code#element.code\
+        @values#element.values[0]\
+        @statusCode#element.statusCode
+            </code></pre>
+        </p>
+        <h3>References</h3>
+        <p>
+            These mappings are defined as per the implementation guide -<a href="http://www.hl7.org/documentcenter/public/standards/dstu/CDAR2_IG_IHE_CONSOL_DSTU_R1dot1_2012JUL.zip">HL7 Implementation Guide for CDA® Release 2</a></br>
+</br>
+            CCDA sample file used in this bundle for testing is from <a href="https://github.com/chb/sample_ccdas/blob/master/Cerner%20Samples/Transition_of_Care_Referral_Summary.xml">Cerner Sample - Transition of Care Referral Summary</a>
```

Comment:
Since the Cerner test file isn't included in the testing anymore, need to remove this from the additionalDetails.html